python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -533,6 +557,11 @@ async def end_disable(self, id_data):
         disable.disableAll = True
         self.mark_corrections(disable, False)
 
+        # Note that the mirror should only be lowered when
+        # coming from the ENABLED state.
+        if self.lower_mirrors_on_disable_state_transition:
```

Comment:
The `begin` and `end` `disable` methods are called in response to a `disable` command so they are only called  when going from `ENABLED` to `DISABLE`... 

That probably means this flag is not necessary.

Reply:
crap, i addressed this before and removed it but didn't commit it :(